src/opm/parser/eclipse/EclipseState/Schedule/ActionAST.cpp

```diff
@@ -318,9 +318,8 @@ ASTNode ActionAST::parse_or(ActionParser& parser) {
 
 ActionAST::ActionAST(const std::vector<std::string>& tokens) {
     ActionParser parser(tokens);
-    auto current = parser.next();
```

Comment:
Ehhh - the `parser.next()` mutates the `parser` object; this will not work - I think ...

Reply:
okay, so we parser.next() but don't catch the result then.

Reply:
> okay, so we parser.next() but don't catch the result then.

That is fine.

Reply:
Ehh - now it is `parser.current()` - we need `parser.next()` to actually advance the internal state of the parser object before the `parse_or()` - I hope this fails on the tests - otherwise I have failed in the testing - so to say!

Reply:
ehm, yeah, mixed em up.

Reply:
test picked it up fine, force pushed.